VsCredentialProvider/Sample.VisualStudio.CredentialProvider/SampleCredentialProvider.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.VisualStudio;
+
+namespace Sample.VisualStudio.CredentialProvider
+{
+
+    [Export(typeof(IVsCredentialProvider))]
+    public sealed class SampleCredentialProvider
+        : IVsCredentialProvider
+    {
+        /// <summary>
+        /// Get credentials for the supplied package source Uri.
+        /// </summary>
+        /// <param name="uri">The NuGet package source Uri for which credentials are being requested. Implementors are
+        /// expected to first determine if this is a package source for which they can supply credentials.
+        /// If not, then Null should be returned.</param>
+        /// <param name="proxy">Web proxy to use when comunicating on the network.  Null if there is no proxy
+        /// authentication configured.</param>
+        /// <param name="isProxyRequest">True if if this request is to get proxy authentication
+        /// credentials. If the implementation is not valid for acquiring proxy credentials, then
```

Comment:
Is this referring to the implementation of the method or the implementation of the package source?

If the first, I would use `does not support` instead of `is not valid for`, if the latter, I would move the statement into the `<returns>`.